prow/jobs/kyma/tests/end-to-end/upgrade-e2e/upgrade-e2e-tests-generic.yaml

```diff
@@ -0,0 +1,47 @@
+# Code generated by rendertemplates. DO NOT EDIT. 
+
+
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+job_template: &job_template
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  labels:
+    preset-dind-enabled: "true"
+    preset-sa-gcr-push: "true"
+    preset-docker-push-repository-kyma: "true"
+  extra_refs:
+    - <<: *test_infra_ref
+      base_ref: master
+  branches:
+    - ^master$
+    - ^release-(1\.13|1\.12|1\.11)-upgrade-e2e$
```

Comment:
This is not good. The job is not valid for older releases. It gets introduced in 1.14 for the first time.